src/utils.js

```diff
@@ -5,14 +5,12 @@ const { readFile, writeFile } = require('fs');
 const Promise = require('bluebird');
 const fetch = require('node-fetch');
 const mkdir = require('mkdirp');
-const { name } = require('../package');
+const pkg = require('../package');
 
 const mkdirp = Promise.promisify(mkdir);
 const writer = Promise.promisify(writeFile);
 const reader = Promise.promisify(readFile);
 
-const headers = { 'User-Agent': name };
```

Comment:
I thought it made more sense to have a global `headers` object. As base headers values grow, we don't want that re-instantiating on every function call. Could "be lazy" & reuse the same object.